proto/protocolapis/astria/protocol/genesis/v1alpha1/types.proto

```diff
@@ -15,6 +17,7 @@ message GenesisAppState {
   IbcParameters ibc_parameters = 8;
   repeated string allowed_fee_assets = 9;
   Fees fees = 10;
+  SlinkyGenesis slinky = 11;
```

Comment:
I was thinking using "slinky" everywhere (at least in all user-facing places like configs and protos) rather than the more generic "oracle" might be more future-proof in case we add more oracles down the line?  Or maybe we could use "slinky_oracle" or something to help readers unaware of slinky?

Reply:
Yeah, I have changed my mind. We should probably call this slinky because there is nothing generic about this. (Or should we use their new `*-connect` name already?)